src/components/header/Header.jsx

```diff
@@ -18,7 +18,11 @@ const Header = () => {
           <ButtonPill text="Log in" onClick={logInOnClick} />
         </div>
       ) : (
-        <p className="ml-auto content-end text-white">I'm logged in!!!!</p>
+        <img
+          className="ml-auto content-end rounded-full"
+          src={rootStore.userStore.userProfileImageUrl}
+          style={{ height: 20, width: 20 }}
+        />
```

Comment:
Step 7: Last but not least, we need to get the image data and then set that as `src` attribute for a new `<img />` element to show the users profile picture.

This is a very hacked up solution and not finalized. It's just there to demonstrate to you how we can get a users profile picture showing!